src/graphviz-1-remove-decl.patch

```diff
@@ -0,0 +1,398 @@
+diff -ur a/ast_common.h b/ast_common.h
```

Comment:
Patches should be in git format-patch style (like `graphviz-2-remove-regex.patch` below) instead of diff style